paparazzi/api/paparazzi.api

```diff
@@ -0,0 +1,255 @@
+	public final fun getNightMode ()Lcom/android/resources/NightMode;
+	public final fun getOrientation ()Lcom/android/resources/ScreenOrientation;
+	public final fun getRatio ()Lcom/android/resources/ScreenRatio;
+	public final fun getReleased ()Ljava/lang/String;
+	public final fun getScreenHeight ()I
+	public final fun getScreenRound ()Lcom/android/resources/ScreenRound;
+	public final fun getScreenWidth ()I
+	public final fun getSize ()Lcom/android/resources/ScreenSize;
+	public final fun getSoftButtons ()Z
+	public final fun getTouchScreen ()Lcom/android/resources/TouchScreen;
+	public final fun getUiMode ()Lcom/android/resources/UiMode;
+	public final fun getUiModeMask ()I
+	public final fun getXdpi ()I
+	public final fun getYdpi ()I
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public final class app/cash/paparazzi/DeviceConfig$Companion {
+	public final fun getEnumMap (Ljava/io/File;)Ljava/util/Map;
+	public final fun loadProperties (Ljava/io/File;)Ljava/util/Map;
+}
+
+public final class app/cash/paparazzi/Environment {
+	public static final field $stable I
+	public fun <init> (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;ILjava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;)V
+	public final fun component1 ()Ljava/lang/String;
+	public final fun component10 ()Ljava/util/List;
+	public final fun component2 ()Ljava/lang/String;
+	public final fun component3 ()Ljava/lang/String;
+	public final fun component4 ()I
+	public final fun component5 ()Ljava/util/List;
+	public final fun component6 ()Ljava/util/List;
+	public final fun component7 ()Ljava/util/List;
+	public final fun component8 ()Ljava/util/List;
+	public final fun component9 ()Ljava/util/List;
+	public final fun copy (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;ILjava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;)Lapp/cash/paparazzi/Environment;
+	public static synthetic fun copy$default (Lapp/cash/paparazzi/Environment;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;ILjava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;ILjava/lang/Object;)Lapp/cash/paparazzi/Environment;
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getAllModuleAssetDirs ()Ljava/util/List;
+	public final fun getAppTestDir ()Ljava/lang/String;
+	public final fun getCompileSdkVersion ()I
+	public final fun getLibraryAssetDirs ()Ljava/util/List;
+	public final fun getLibraryResourceDirs ()Ljava/util/List;
+	public final fun getLocalResourceDirs ()Ljava/util/List;
+	public final fun getModuleResourceDirs ()Ljava/util/List;
+	public final fun getPackageName ()Ljava/lang/String;
+	public final fun getPlatformDir ()Ljava/lang/String;
+	public final fun getResourcePackageNames ()Ljava/util/List;
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public final class app/cash/paparazzi/EnvironmentKt {
```

Comment:
Is there a way to make these `@JvmStatic` on `Environment` for Java consumers? Probably breaking for Kotlin users due to import change.

Reply:
they're top level functions, so Java consumers can already access them like so:

```
import static app.cash.paparazzi.EnvironmentKt.detectEnvironment;

import app.cash.paparazzi.Environment;

public class Foo {
  Environment env = detectEnvironment();
}
```

Reply:
Yeah, it's just leaking the implementation details of Kotlin (Kt suffix)